### PR TITLE
Binod

### PR DESCRIPTION
**Pull request by donib-irakihda**

### CODE REVIEW

File: controller/adminController.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+      }
+    }
+  } catch (error) {
+    console.log(error);
+    res.status(500).json({ error: "Something went wrong while logging in" });
+  }
+};
+
+const onlyAdmin = async (req, res) => {
+  console.log("I got here");
```
Comment (IamAbishan1) [inline controller/adminController.js:30]: remove this console here

Comment (Sandesh1998) [inline controller/adminController.js:30]: Remove console.log

File: controller/feedbackController.js [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+    }
+
+    if (
+      feedback.user.toString() !== req.user._id.toString() &&
+      req.user.userType !== "admin"
+    ) {
+      return res.status(401).json({ message: "Not authorized" });
+    }
+
+    const newFeedback = {
```
Comment (IamAbishan1) [inline controller/feedbackController.js:74]: check this looks like not updated in db

File: controller/userController.js [hunk 3]
```diff
@@ -126,29 +113,41 @@ const loginUser = async (req, res) => {
...
-    res.status(200).json({ message: "Logout success" });
-  }
-  catch (error) {
-    console.log(error);
-    res.status(500).json({ error: "Logout failed" });
-  }
-};
-
-module.exports = { registerUser, loginUser, logoutUser };
+// const logoutUser = async (req, res) => {
```
Comment (Sandesh1998) [inline controller/userController.js:138]: Remove commented code before you push the code

File: middleware/auth.js [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+      }
+    } else {
+      return res.status(500).json({ error: "Token not found" });
+    }
+  } catch (error) {
+    console.error("Something went wrong", error);
+  }
+};
+
+// const isLoggedOut = async (req, res, next) => {
```
Comment (Sandesh1998) [inline middleware/auth.js:23]: remove commented code here too

**Review by IamAbishan1 — APPROVED** [review]
👍

**Pull request merged**